src/main/resources/anniv6Resources/localization/eng/BeastsLair/Powerstrings.json

```
@@ -0,0 +1,15 @@
+{
+
+  "${ModID}:FuryPower": {
+    "NAME": "Fury",
+    "DESCRIPTIONS": [
+      "Whenever you play a card, gains #b", "% Damage up until end of turn."
```

I'd word this the same way as the Damage Up power. I think that's clearer than saying that it gains Damage Up
```suggestion
      "Whenever you play a card, deals #b", "% more damage until end of turn."
```